docs/docs/dev-setup.md

```diff
@@ -166,15 +181,23 @@ We suggest two main [options](https://docs.zephyrproject.org/latest/guides/env_v
 
 To load the Zephyr environment properly for just one transient shell, run the following from your ZMK checkout directory:
 
+Unix:
```

Comment:
Not sure if this is the best way to split up Windos/Unix. Would be nice to have tabs to click on like in the Zephyr docs.